index.js

```diff
@@ -358,6 +365,11 @@ input UndeleteSectionInput {
   id: ID!
 }
 
+input DuplicateSectionInput {
+  id: ID!
+  position: Int!
```

Comment:
How come we need to provide position here? Can't we get it from the page/section view? Or is that position the target position for the copy?

Reply:
It's the target position for the copy.